can/can_graph.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python3
+# *_* coding: utf-8 *_*
+
+"""
+Creates and runs the same graph as the babeltrace-fun-plugins/can/check.sh utility.
+---
+Please note: libbabeltrace2 python library (bt2) depends on its core C library.
```

Comment:
I would write this with the perspective that babeltrace will be `make install`'ed somewhere.  Running inside the build directory is kind of a corner case of babeltrace developers, whereas these examples are more meant for users.

So if babeltrace has been installed system-wide (e.g. in `/usr`), there is generally nothing to set.  But if it's been installed to some random location (e.g. in `/opt/babeltrace`), then you would need to set some variables.  Could you please adapt the explanations below to that scenario?

In my experience, it is necessary to set `LD_LIBRARY_PATH` and `PYTHONPATH`.  Did you also need to set `PYTHONPATH`?

So it would be:

    LD_LIBRARY_PATH=[babeltrace2 install directory]/lib
    PYTHONPATH=[babeltrace2 install directory]/lib/[python version]/site-packages

Reply:
Actually, maybe this info should go into the top-level `README.md`, since it will always be the same.